src/server.js

```diff
@@ -40,6 +41,10 @@ module.exports = function(options) {
 	}
 
 	if ((config.identd || {}).enable) {
+		if (manager.identHandler) {
+			log.warn("Warning: using both identd and oidentd at the same time.");
```

Comment:
Do we need to specify `Warning: ` if this is already a `log.warn`? Also, considering the content, I'd go for an info rather than a warning.